packages/jaeger-ui/src/components/SearchTracePage/SearchResults/index.js

```diff
@@ -167,6 +170,7 @@ export class UnconnectedSearchResults extends React.PureComponent<SearchResultsP
                   traceID: t.traceID,
                   size: t.spans.length,
                   name: t.traceName,
+                  color: t.spans.some(sp => sp.tags.some(isErrorTag)) ? 'red' : '#12939A',
```

Comment:
Can we use constants for the colors?

Reply:
Can you advise me on how we do this elsewhere?  For example https://github.com/jaegertracing/jaeger-ui/blob/main/packages/jaeger-ui/src/components/DependencyGraph/DAG.js#L80 uses them inline, so I did as well.

The shade of blue we use, #12939A, seems to come from https://github.com/uber/react-vis but I couldn't actually figure out if it was coming from .css or .js, and which file.